include/dp_session.h

```diff
@@ -65,6 +65,63 @@ enum dp_session_state {
 	SESSION_STATE_CLOSED,
 } __attribute__ ((__packed__));
 
+/**
```

Comment:
It would seem more sensible to have these as part of a session feature block, in the same way as the "npf" session lives within a session feature block.  Look at src/npf/npf_session.c, it being the only other real user of the current DP_SESSION.

So why can't this be a session feature block?

Reply:
This dp_session_attr and dp_session_info needed by dp_session_query. The dp_session_query is a public function (which intend be called by plugin) like dp_session_unique_id, dp_session_get_state, etc... but instead of getting one field it get all necessary fields by `query` parameter.

Most of fields could be fetched from a session but dpi info, to get dpi info I need access npf_session, thus I take idea from session_log. Simply adding a `query` operation into npf session_feature_ops then invoke it.

Also, the `query` operation work mostly like `json` operation except it return raw info not formatted json string. Note that the flowstat plugin need raw info to process in it own purpose, not the json string one.